stdlib/public/core/DropWhile.swift

```diff
@@ -46,52 +74,21 @@ public struct LazyDropWhileIterator<Base : IteratorProtocol> :
       }
     }
     return nil
-  }
-
-  @_inlineable // FIXME(sil-serialize-all)
-  @_versioned // FIXME(sil-serialize-all)
-  internal init(_base: Base, predicate: @escaping (Base.Element) -> Bool) {
-    self._base = _base
-    self._predicate = predicate
-  }
-
-  @_versioned // FIXME(sil-serialize-all)
-  internal var _predicateHasFailed = false
-  @_versioned // FIXME(sil-serialize-all)
-  internal var _base: Base
-  @_versioned // FIXME(sil-serialize-all)
-  internal let _predicate: (Base.Element) -> Bool
+  }  
 }
 
-/// A sequence whose elements consist of the elements that follow the initial
-/// consecutive elements of some base sequence that satisfy a given predicate.
-@_fixed_layout // FIXME(sil-serialize-all)
-public struct LazyDropWhileSequence<Base : Sequence> : LazySequenceProtocol {
-
-  public typealias Elements = LazyDropWhileSequence
-
+extension LazyDropWhileSequence: Sequence {
```

Comment:
Since `SubSequence` is part of `Sequence` wouldn't it make sense to declare it here?

Reply:
Yeah I didn't because it gives me a sad, because it is `AnySequence`... I guess I should.